contracts/QuestChain.sol

```diff
@@ -365,10 +354,11 @@ contract QuestChain is
 
     function mintToken() external {
         require(questCount > 0, "QuestChain: no quests found");
-        for (uint256 questId = 0; questId < questCount; questId = questId + 1) {
+        for (uint256 _questId; _questId < questCount; ++_questId) {
```

Comment:
What if all the quests are either optional or paused, do we still want to allow minting? @dan13ram